dev-academy/develop-smart-contract/02-develop.md

```diff
@@ -176,211 +146,483 @@ pub fn execute(
     msg: ExecuteMsg,
 ) -> Result<Response, ContractError> {
     match msg {
-        ExecuteMsg::CreatePot {
-            target_addr,
-            threshold,
-        } => execute_create_pot(deps, info, target_addr, threshold),
-        ExecuteMsg::Receive(msg) => execute_receive(deps, info, msg),
+        ExecuteMsg::NewEntry {description, priority} => create_new_entry(deps, info, description, priority),
```

Comment:
For function names, best to follow best practices. you should use `action_create...` naming convention